solr/contrib/ltr/src/java/org/apache/solr/ltr/norm/StandardNormalizer.java

```diff
@@ -69,6 +70,13 @@ public void setStd(String std) {
   public float normalize(float value) {
     return (value - avg) / std;
   }
+  
+  @Override
+  protected void validate() throws ModelException {
+    super.validate();
+    if (std <= 0f)
+      throw new ModelException("Standard Normalizer standard deviation must be positive");
```

Comment:
Suggest including the `avg` and `std` values in the exception since that will make it easier for users to locate the .json portion that needs changing.